notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<p>When you start to compute values in line 42 (and onwards), it was a bit hard to follow how you created the results list with the quantities defined in the data frame (line 6). The hardest part to follow was actually the "vect = [ hue list ]" on line 71. Maybe breaking this in some functions? Or simply adding more comments? I don't now, but it was a bit hard to see exactly what is happening.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I agree, the logic is difficult to follow. </p><p>The cell has been re-arranged, thanks to lab_black and I have added comments. I hope that it is clearer.</p>